### PR TITLE
Drop psql native bindings - Closes #664 #1149

### LINKED ISSUE #664: Drop PostgreSQL native bindings
Would be nice to drop `pg-native`, however research should be done how much it will affect performance.

### PR DESCRIPTION
**Pull request by diego-G**
### Review checklist

* The PR solves #664 and likely #1149 to confirm in QA phase.
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

**Comment by vitaly-t** [comment]
You've never followed up on version 1.0.0

Did you manage to reproduce the same issue as in 0.9.12?

**Comment by diego-G** [comment]
I've done the follow up as far as I was able to. We will do pertinent tests in QA phase within several nodes. But you are right, I add the second connected issue to the list of fixes.

File: db/index.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ const repos = require('./repos');
 // TODO: Had to change below from 'const' into 'let' because of the nasty 'rewire' hacks inside DBSandbox.js.
 // eslint-disable-next-line prefer-const
 let initOptions = {
-	pgNative: true,
```
Comment (4miners) [inline db/index.js]: default is `false`, we can remove entirely

Comment (diego-G) [reply db/index.js]: I'd prefer to leave it until we start QA testing.

Comment (4miners) [reply db/index.js]: kind of pointless, because we removed dependency, but as you wish ;)

**Pull request merged**